modules/new_serverpod_auth/serverpod_auth_email/serverpod_auth_email_server/lib/src/endpoints/email_account_endpoint.dart

```diff
@@ -111,16 +133,21 @@ abstract class EmailAccountEndpoint extends Endpoint {
 
```

Comment:
This did not use a transaction on purpose, as the "destroy all sessions" has that major side-effect of and thus it seemed unsuitable to roll back (and also there is no bad "partial" stuff being done).

But I also don't have a super strong/argument feeling against this, so maybe it'd be fine to add it for consistency.